### DISCUSSION TITLE
Citrix Receiver not working?

### DISCUSSION THREAD

**Discussion by xTITUSMAXIMUSX** [ltsp/ltsp#262] [Q&A]
The main goal I am trying to achieve is to run citrix receiver on the clients. It will run just fine if you run it as root but I'm not sure why it wont run as a regular user. I'm not really seeing any logs being generated to provide any more insight.

**Comment by alkisg** [comment]
This doesn't sound related to LTSP...
If you clone the image to a local disk and boot from local disk, does it work then?

I've only used citrix back in 2007 before moving to LTSP, so I've no idea how it works; I would need to check over VNC.
